cmd/inspect.go

```diff
@@ -94,11 +101,35 @@ func printDeps(deps map[string][]string, tree bool, ip gpsr.InstallPlan) {
 	}
 }
 
+func installedFromPackages() {
+	var pkgr, packrat, unknown []string
+	ip := GetInstalledPackages()
+	for k, v := range ip {
+		if len(v.PkgrVersion) == 0 {
+			packrat = append(packrat, k)
```

Comment:
why packrat if PkgrVersion not present? 
Any reason not to do v.PkgrVersion == "", why check length?

Reply:
Interpreted from this statement:  "with the definition of not installed by pkgr being the metadata not detected in description file"

no reason for len, I think it did not compile the first time when using ==, probably due to an unrelated syntax error